SI/Sem.03/Pract.03/Таsк06.cpp

```diff
@@ -0,0 +1,27 @@
+#include <iostream>
+int main() {
+	int num = 0, count = 0;
+	do {//checks if the number is possitive or 0
+		std::cin >> num;
+		if (num < 0) {
+			std::cout << "Insert positive number: " << std::endl;
+		}
+		if (num == 0) break;
+	} while (num < 0);
+	//all numbers two times, except for 0
+	while (true) {
+		std::cin >> num;
+		if (num == 0) {
+			std::cout << num << std::endl;
+			break;
+		}
+		count++;
+		if (count <= 2) {
+			std::cout << "Insert again " << num << "." << std::endl;
+		}
+		else {
+			count = 0;
+		}
+		
+	}
+}
```

Comment:
You probably misunderstood the task, so you will need to rewrite it. The user is never supposed to enter zero. You have to find the missing number in a sequence.